whiskey/public/data/whiskies.json

```diff
@@ -3171,6 +3171,136 @@
         "Sweet"
       ],
       "average_rating": 71
-    }
-  ]
+    },
+  {
+      "age":10,
+      "brand": "Wild Turkey",
+      "price":130,
+      "event_number":31,
+      "event_date": "10/11/2017",
+      "image_url": "",
+      "name": "Master's Keep Decades",
+      "origin": "USA",
+      "type": "Bourbon",
+      "url": "http://wildturkeybourbon.com/about/our-products/",
+      "description": "Caramel and mint on the nose. Followed by robust vanilla, toasted oak, and spices",
+      "ratings":[
```

Comment:
Generally, I alphabetize these, but it's just a stylistic preference. Currently, the front-end doesn't alphabetize them on the modals, but I could add that.